variants/esp32c6/m5stack_unitc6l/variant.cpp

```diff
@@ -0,0 +1,74 @@
+#include "driver/gpio.h"
+#include <Arduino.h>
+#include <Wire.h>
+// I2C device addr
+#define PI4IO_M_ADDR 0x43
+
+// PI4IO registers
+#define PI4IO_REG_CHIP_RESET 0x01
+#define PI4IO_REG_IO_DIR 0x03
+#define PI4IO_REG_OUT_SET 0x05
+#define PI4IO_REG_OUT_H_IM 0x07
+#define PI4IO_REG_IN_DEF_STA 0x09
+#define PI4IO_REG_PULL_EN 0x0B
+#define PI4IO_REG_PULL_SEL 0x0D
+#define PI4IO_REG_IN_STA 0x0F
+#define PI4IO_REG_INT_MASK 0x11
+#define PI4IO_REG_IRQ_STA 0x13
+// PI4IO
+
+#define setbit(x, y) x |= (0x01 << y)
+#define clrbit(x, y) x &= ~(0x01 << y)
+#define reversebit(x, y) x ^= (0x01 << y)
+#define getbit(x, y) ((x) >> (y)&0x01)
+
+void i2c_read_byte(uint8_t addr, uint8_t reg, uint8_t *value)
+{
+    Wire.beginTransmission(addr);
+    Wire.write(reg);
+    Wire.endTransmission();
+    Wire.requestFrom(addr, 1);
+    *value = Wire.read();
+}
+
+/*******************************************************************/
+void i2c_write_byte(uint8_t addr, uint8_t reg, uint8_t value)
+{
+    Wire.beginTransmission(addr);
+    Wire.write(reg);
+    Wire.write(value);
+    Wire.endTransmission();
+}
+/*******************************************************************/
+void c6l_init()
+{
+    // P7 LoRa Reset
+    // P6 RF Switch
```

Comment:
As pointed out bove, this is prime initLateVariant() stuff

Reply:
I tried. Doing this in initLateVariant() causes the LoRa radio to stop responding. Likely because the reset line gets twiddled while setting up the GPIO extender.